Fix Issue 7775 - duplicated text in make-calls.rst (#7776)
mattermost/docs: source/collaborate/make-calls.rst

.. tab:: Web/Desktop

>   Open the chat thread in the widget by selecting |gear| and there click on "Show chat thread". Alternative expand the call window using the arrows in the top-right of the call widget. From there, select the chat icon to access the chat thread.

Reviewer: ```suggestion
  Open the chat thread in the widget by selecting the **Gear** |gear| icon, then select **Show chat thread**. Alternatively, expand the call window using the arrows in the top-right of the call widget. From there, select the chat icon to access the chat thread.
```
